### DISCUSSION TITLE
Data filtered in table or in inspect data are not filtered to Download CSV

### DISCUSSION THREAD

**Discussion by sdelobel** [grafana/grafana#41540] [Dashboards, Panels & Visualizations]
**What happened**: The data filtered in table and/or in inspect data are not filtered when we donwload CSV

**What you expected to happen**: The data filtered in table and/or in inspect data are filtered when we donwload CSV

**How to reproduce it (as minimally and precisely as possible)**:
Create a table with any datas

Set Column filter in this table
Set a filter in one column
![1](https://user-images.githubusercontent.com/16326301/140985433-f14d1be7-2613-43ec-b510-7ae8e2db0091.png)
Inspect the Data (the data are not filtered)
![2](https://user-images.githubusercontent.com/16326301/140985435-83d59580-1779-4593-a9f3-28b98faedea7.png)
![3](https://user-images.githubusercontent.com/16326301/140985436-c6a22b36-1d06-4dc9-a122-d9c3270f9626.png)
Set a filter in one column (again) in Inspect Data panel
![4](https://user-images.githubusercontent.com/16326301/140985427-fe16970d-e988-4797-9205-6eddba77459a.png)
Download CSV
![5](https://user-images.githubusercontent.com/16326301/140985437-7480404a-2cf9-4ac1-9bbc-80b571c4edcb.png)
And the data downloaded are not filtered

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.2.3
- Data source type & version: Postgres 12.0
- OS Grafana is installed on: Centos 7
- User OS & Browser: MacOS & Firefox

- Grafana plugins:
- Others:

**Comment by torkelo** [comment]
the download CSV is for the full data set , the filter is only for the view

**Reply by sdelobel** [reply]
Ok, it's not a bug, but can you setting up this features ?

**Reply by sdelobel** [reply]
cf #29113

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
